admin/taxonomy/class-taxonomy-metabox.php

```diff
@@ -244,76 +235,9 @@ private function has_single_social_tab() {
 	 */
 	private function get_buy_premium_link() {
 		return sprintf(
-			"<div class='%s'><a href='#wpseo-meta-section-premium' class='wpseo-meta-section-link'><span class='dashicons dashicons-star-filled wpseo-buy-premium'></span>%s</a></div>",
-			'wpseo-metabox-buy-premium',
-			__( 'Go Premium', 'wordpress-seo' )
-		);
-	}
-
-	/**
-	 * Returns the metabox section for the Premium section..
-	 *
-	 * @return WPSEO_Metabox_Section
-	 */
-	private function get_buy_premium_section() {
-		$content = sprintf(
-			"<div class='wpseo-premium-description'>
-				%s
-				<ul class='wpseo-premium-advantages-list'>
-					<li>
-						<strong>%s</strong> - %s
-					</li>
-					<li>
-						<strong>%s</strong> - %s
-					</li>
-					<li>
-						<strong>%s</strong> - %s
-					</li>
-					<li>
-						<strong>%s</strong> - %s
-					</li>
-				</ul>
-	
-				<a target='_blank' id='wpseo-buy-premium-popup-button' class='button button-buy-premium wpseo-metabox-go-to' href='%s'>
-					%s
-				</a>
-	
-				<p><a target='_blank' class='wpseo-metabox-go-to' href='%s'>%s</a></p>
-			</div>",
-			/* translators: %1$s expands to Yoast SEO Premium. */
-			sprintf( __( 'You\'re not getting the benefits of %1$s yet. If you had %1$s, you could use its awesome features:', 'wordpress-seo' ), 'Yoast SEO Premium' ),
-			__( 'Redirect manager', 'wordpress-seo' ),
-			__( 'Create and manage redirects within your WordPress install.', 'wordpress-seo' ),
-			__( 'Synonyms & related keyphrases', 'wordpress-seo' ),
-			__( 'Optimize a single post for synonyms and related keyphrases.', 'wordpress-seo' ),
-			__( 'Social Previews', 'wordpress-seo' ),
-			__( 'Check what your Facebook or Twitter post will look like.', 'wordpress-seo' ),
-			__( 'Premium support', 'wordpress-seo' ),
-			__( 'Gain access to our 24/7 support team.', 'wordpress-seo' ),
-			WPSEO_Shortlinker::get( 'https://yoa.st/pe-buy-premium' ),
-			/* translators: %s expands to Yoast SEO Premium. */
-			sprintf( __( 'Get %s', 'wordpress-seo' ), 'Yoast SEO Premium' ),
-			WPSEO_Shortlinker::get( 'https://yoa.st/3g5' ),
-			__( 'More info', 'wordpress-seo' )
-		);
-
-		$tab = new WPSEO_Metabox_Form_Tab(
-			'premium',
-			$content,
-			'Yoast SEO Premium',
-			array(
-				'single' => true,
-			)
-		);
-
-		return new WPSEO_Metabox_Tab_Section(
-			'premium',
-			'<span class="dashicons dashicons-star-filled wpseo-buy-premium"></span>',
-			array( $tab ),
-			array(
-				'link_aria_label' => 'Yoast SEO Premium',
-				'link_class'      => 'yoast-tooltip yoast-tooltip-e',
-			)
+			'<div class="wpseo-metabox-buy-premium"><a target="_blank" rel="noopener noreferrer" href="%1$s"><span class="dashicons dashicons-star-filled wpseo-buy-premium"></span>%2$s</a></div>',
```

Comment:
As per the new pattern for links pointing to yoast.com, ` rel="noopener noreferrer"` should be removed.

Reply:
Also, it would be nice to add the `(Opens in a new browser tab)` message to this link both in the taxonomy and post meta box.